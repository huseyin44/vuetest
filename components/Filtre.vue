<template>
<div>
  <v-container class="satirbir" fluid>
    <v-row contain>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Kategori
            </div>
            </template>
            <v-list class="kategorilist" >
             <div class=" red14" >Saat</div>
             <div class=" red14">Fırsat</div>
             <div class=" red14">Online-Mağazaya-Özel</div>
             <div class=" red14" >İndirimli-Saatler </div>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col>
       <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1" >
             Cinsiyet
            </div>
            </template>
            <v-list class="kategorilist" >
             <v-radio class=" red14" label="Erkek"></v-radio>
             <v-radio class=" red14" label="Kadın"></v-radio>
            </v-list>
         </v-menu>
       </v-row>
      </v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1" >
             Fiyat
            </div>
            </template>
            <v-list class="kategorilist" >
             <div class=" red14">1,000,000TL-1,498,000TL</div>
             <div class=" red14">1,500,000TL-1,998,000TL</div>
             <div class=" red14">2,000,000TL-2,498,000TL</div>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Renk
            </div>
            </template>
             <v-list class="kategorilist" >
             <v-radio class=" red14" label="Metalik Gri"></v-radio>
             <v-radio class=" red14" label="Rose Gold"></v-radio>
             <v-radio class=" red14" label="Siyah"></v-radio>
             <v-radio class=" red14" label="Kahverengi"></v-radio>
             <v-radio class=" red14" label="Beyaz"></v-radio>
             <v-radio class=" red14" label="Pempe"></v-radio>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Kayış Tipi
            </div>
            </template>
             <v-list class="kategorilist" >
              <v-radio class=" red14" label="Çelik"></v-radio>
              <v-radio class=" red14" label="Deri"></v-radio>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Tarz
            </div>
            </template>
             <v-list class="kategorilist" >
              <v-radio class=" red14" label="Spor Saatler"></v-radio>
              <v-radio class=" red14" label="Klasik Saatler"></v-radio>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Teknoloji
            </div>
            </template>
             <v-list class="kategorilist" >
              <v-radio class=" red14" label="Quartz"></v-radio>
              <v-radio class=" red14" label="Otomatik"></v-radio>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col class="buton1">Urun</v-col>
      <v-col class="buton1"></v-col>
      <v-col class="buton1"></v-col>
      <v-col class="buton1"></v-col>
      <v-col class="buton1"></v-col>
      <v-col class="buton1"></v-col>
      <v-col class="buton1"></v-col>
      <v-col>
        <v-row>
          <v-menu  open-on-hover offset-y  >
           <template v-slot:activator="{ on, attrs }">
            <div v-bind="attrs" v-on="on" class="buton1">
             Sıralama
            </div>
            </template>
             <v-list class="kategorilist" >
              <v-radio class=" red14" label="Seçtiklerimiz"></v-radio>
              <v-radio class=" red14" label="Fiyat Artan"></v-radio>
              <v-radio class=" red14" label="Fiyat Azalan"></v-radio>
              <v-radio class=" red14" label="Yeni Gelenler"></v-radio>
              <v-radio class=" red14" label="İndirimler"></v-radio>
            </v-list>
         </v-menu>
        </v-row>
      </v-col>
      <v-col class="buton1"></v-col>
    </v-row>
  </v-container>
</div>
</template>

<script>
export default {
 
}
</script>

<style>
.satirbir{
  background: #F9F9F9;
  height: 45px;
  max-height: 45px;
}
.buton1{
  background: #F9F9F9;
  max-width: 100px;
  height:20px;
  white-space: nowrap;
  color: #353535;
  font-size: 16px;
  min-height:0px
}
.red14:hover {
  background: gray;
  color: black;
}
.buton1:hover{
  background: white;
  color: #BD1818;
}
</style>