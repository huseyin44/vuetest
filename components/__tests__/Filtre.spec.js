import { shallowMount } from "@vue/test-utils";
import Filtre from "../Filtre";
import Vue from 'vue'
import Vuetify from 'vuetify'
Vue.use(Vuetify)
const factory = () => {
    return shallowMount(Filtre, {
    });
  };
describe('Filtre', () => {
    let vuetify
    beforeEach(() => {
      vuetify = new Vuetify()
    })
  
    it('Bir Vue kopmonenti', () => {
        const wrapper = factory();
        expect(wrapper.isVueInstance()).toBeTruthy();
    })
    it('Snapshot Ozelligi', () => {
        const wrapper = factory();
        expect(wrapper.html()).toMatchSnapshot();
    })
    test('triggers a mouseover', async () => {
        const wrapper = shallowMount(Filtre)
        await wrapper.trigger('mouseover')
      })
    test('triggers a mouseleave', async () => {
        const wrapper = shallowMount(Filtre)
        await wrapper.trigger('mouseleave')
      })
    
  })