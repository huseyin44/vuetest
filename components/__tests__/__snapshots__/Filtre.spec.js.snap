// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filtre Snapshot Ozelligi 1`] = `
<div>
  <v-container-stub tag="div" fluid="true" class="satirbir">
    <v-row-stub tag="div" contain="">
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <div class=" red14">Saat</div>
              <div class=" red14">Fırsat</div>
              <div class=" red14">Online-Mağazaya-Özel</div>
              <div class=" red14">İndirimli-Saatler </div>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Erkek" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Kadın" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <div class=" red14">1,000,000TL-1,498,000TL</div>
              <div class=" red14">1,500,000TL-1,998,000TL</div>
              <div class=" red14">2,000,000TL-2,498,000TL</div>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Metalik Gri" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Rose Gold" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Siyah" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Kahverengi" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Beyaz" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Pempe" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Çelik" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Deri" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Spor Saatler" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Klasik Saatler" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Quartz" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Otomatik" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div" class="buton1">Urun</v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
      <v-col-stub tag="div">
        <v-row-stub tag="div">
          <v-menu-stub opendelay="0" closedelay="0" contentclass="" openonhover="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" openonclick="true" closeonclick="true" closeoncontentclick="true" maxheight="auto" offsety="true" origin="top left" transition="v-menu-transition">
            <v-list-stub tag="div" class="kategorilist">
              <v-radio-stub ripple="true" label="Seçtiklerimiz" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Fiyat Artan" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Fiyat Azalan" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="Yeni Gelenler" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
              <v-radio-stub ripple="true" label="İndirimler" officon="$radioOff" onicon="$radioOn" class=" red14"></v-radio-stub>
            </v-list-stub>
          </v-menu-stub>
        </v-row-stub>
      </v-col-stub>
      <v-col-stub tag="div" class="buton1"></v-col-stub>
    </v-row-stub>
  </v-container-stub>
</div>
`;
